ark/utils/deepcell_service_utils.py

```diff
@@ -0,0 +1,100 @@
+from twisted.internet import reactor
+from kiosk_client import manager
+import os
+import glob
+from zipfile import ZipFile
+import warnings
+
+
+def create_deepcell_output(deepcell_input_dir, points, deepcell_output_dir,
+                           suffix='_feature_0', host='https://deepcell.org', job_type='multiplex'):
+    """ Handles all of the necessary data manipulation for running deepcell tasks.
+
+        Creates .zip files (to be used as input for DeepCell),
```

Comment:
We're using the google style for docstrings. You can see the readthedocs raised an error because of the formatting. You can take a look at the docstrings for some of the other functions in the repo to see what the format is